src/inmanta/moduletool.py

```diff
@@ -768,12 +786,35 @@ def build(
         if output_dir is None:
             output_dir = os.path.join(path, "dist")
 
+        distributions_to_build: Sequence[Literal["wheel", "sdist"]]
+        if distribution is None:
+            distributions_to_build = ["wheel"]
+        elif distribution == "wheel_and_sdist":
+            distributions_to_build = ["wheel", "sdist"]
+        else:
+            distributions_to_build = [distribution]
+
+        timestamp = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
I'm slightly hesitant about this being a naive timestamp. In practice, we currently only pass it to a method that wouldn't even care for the timezone part, but this is counter to the convention we have throughout all our code. So I think I'd prefer to create a tz-aware timestamp here, and then convert to naive UTC in the build method.

Reply:
I don't understand what you mean. The above-mentioned code does provide a timestamp, right?

Reply:
Don't know what I was thinking here, LGTM